src/graphics/aurora/rules.mk

```diff
@@ -49,6 +49,7 @@ src_graphics_aurora_libaurora_la_SOURCES += \
     src/graphics/aurora/animnode.h \
     src/graphics/aurora/animation.h \
     src/graphics/aurora/fadequad.h \
+    src/graphics/aurora/borderquad.h \
```

Comment:
Yeah, the ordering is not really lexicographical. When I added files, I've roughly put them from least specific to more specific, if that makes sense. Base classes before classes that inherit from those. for example.

But that's not really a hard rule. That rule breaks down for some cases, and other people added files in other orders. I'm not really that adamant about it. Not about this, at least ;)